src/storage/db_storage.cpp

```diff
@@ -359,6 +359,26 @@ void DbStorage::addStatusFieldToBatch(StatusDbField const& field, uint64_t const
 }
 
 // PBFT
+uint64_t DbStorage::getPbftMgrPreviousRoundStatus(PbftMgrPreviousRoundStatus const& field) {
+  auto status = lookup(toSlice((uint8_t)field), Columns::pbft_mgr_previous_round_status);
+  if (!status.empty()) {
+    size_t value;
+    memcpy(&value, status.data(), sizeof(uint64_t));
+    return value;
+  }
+
+  return 0;
+}
+
+void DbStorage::savePbftMgrPreviousRoundStatus(PbftMgrPreviousRoundStatus const& field, uint64_t const& value) {
+  insert(Columns::pbft_mgr_previous_round_status, toSlice((uint8_t)field), toSlice(value));
```

Comment:
use static_cast for casting not (uint8_t)

Reply:
Fixed, thanks